dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -127,7 +140,7 @@ static ClassDumper create(List<Path> jarFilePaths) {
       if (!(constantAtNameAndTypeIndex instanceof ConstantNameAndType)) {
         // This constant_pool entry must be a CONSTANT_NameAndType_info
         // as specified https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.4.2
-        throw new RuntimeException(
+        throw new ClassFormatException(
```

Comment:
I read go/java-practices/verification-checks.

@elharo Does "Verify" make more sense to you here?
https://google.github.io/guava/releases/snapshot-jre/api/docs/com/google/common/base/Verify.html

Reply:
I think ClassFormatException is more specific and does fit this case.

Reply:
Good. Then I keep ClassFormatException.